### DISCUSSION TITLE
Official Docs for driver installation of coral.ai dont work on up-to-date linux (in my case Ubuntu). Do i need the driver in the first place ?

### DISCUSSION THREAD

**Discussion by all-hail-the-omnissiah** [blakeblackshear/frigate#14315] [Ask A Question]
Hey, i have a homelab running Proxmox on there is a DockerVM (Ubuntu) and in Docker i run Frigate. Now can i just passthrough the USB Coral TPU to the DockerVM and then put the necessary lines in the compose.yaml and it will work or do i need the drivers that i can't get to work for the hell of it.

**Comment by NickM-27** [comment]
if you are using the m.2 coral then yes the driver is required

**Reply by all-hail-the-omnissiah** [reply]
No i'm using the USB one. So how would i do this ? Just passthrough the driverless device from Proxmox to the DockerVM and then use the lines in the compose.yaml from the Frigate documentation to passthrough the Coral to the container ? And that's it ?

**Reply by NickM-27** [reply]
Then you just need to pass the usb correctly to frigate
